### PR TITLE
[Feature] Add notAssessed to evaluatedLanguageAbility enum

### LINKED ISSUE #7947: New option for language test result: Not assessed
# Details
As an applicant, I want to be able to flag language tests that I have not yet completed, so I can at least enter the ones I have completed.

For Government of Canada language tests, there can often be months between when I do my reading & writing tests, and when I complete the oral exam. Right now, all 3 fields are required, and the options are only for results, so there's no way to submit just 1 or 2 results. Even worse, people are choosing the X options, which technically indicates that they failed the test.

## Proposed solution

Add a new option for the language test results that simply indicates you have not yet completed it:

EN: "Not assessed"
FR: "Non évalué"

## Screenshot
Image

## Acceptance Criteria
- [ ] New option "Not assessed" added to the end of the 3 language test results select inputs.

### PR DESCRIPTION
**Pull request by JamesHuf**
🤖 Resolves #7947

## 👋 Introduction

Adds the new value NOT_ASSESSED to the `EvaluatedLanguageAbility` enum and makes sure it's displayed last in the list of options.

## 🧪 Testing

Assist reviewers with steps they can take to test that the PR does what it says it does.

1. Navigate to the profile `en/users/{id}/profile`
2. Open the language profile form and confirm "Not assessed" shows up at the bottom of the language evaluation level Selects.
3. Save one of the categories as "Not assessed" and confirm it appears correctly in the normal section display.

## 📸 Screenshot

image

### CODE REVIEW

File: apps/web/src/components/UserProfile/ProfileSections/LanguageInformationSection.tsx [hunk 1]
```diff
@@ -159,7 +160,23 @@ const LanguageInformationSection = ({
...
+                    ? intl.formatMessage(
+                        getEvaluatedLanguageAbility(writtenLevel),
+                      )
+                    : "",
+                  verbalLevel
+                    ? intl.formatMessage(
+                        getEvaluatedLanguageAbility(verbalLevel),
+                      )
+                    : "",
+                ].join(", ")}
```
Comment (esizer) [inline apps/web/src/components/UserProfile/ProfileSections/LanguageInformationSection.tsx:179]: I beleive this pattern repeats itself a few times. Would it make sense to move it to a single util function?

**Review by esizer — CHANGES_REQUESTED** [review]
LGTM! 👍 One minor suggestion.

File: apps/web/src/components/UserProfile/ProfileSections/LanguageInformationSection.tsx (hunk 1 above)
Comment (JamesHuf) [reply apps/web/src/components/UserProfile/ProfileSections/LanguageInformationSection.tsx]: Good call! I've created a new util function for this and updated a few places to use it.

**Review by esizer — APPROVED** [review]
Awesome! ⭐

**Pull request merged**